packages/api-explorer/src/components/ExploreType/ExploreProperty.tsx

```diff
@@ -158,7 +158,7 @@ export const ExplorePropertyDescription: FC<ExplorePropertyProps> = ({
 export const ExplorePropertyDetail: FC<ExplorePropertyProps> = ({
   property,
 }) => (
-  <Space>
+  <Space style={{ fontSize: 'small', marginLeft: '10rem' }}>
```

Comment:
Prevents `ExplorePropertyDetail` from covering the label
Bug demo here:
https://drive.google.com/file/d/1c9eHOBJ2utbUKm_Oj8A1u8j2O7m66E7R/view?usp=sharing&resourcekey=0-ZX13bwjqnikSfFQ_WKc3rA